src/ec/packages/ec-component-site-header-harmonised/ecl-site-header-harmonised.html.twig

```diff
@@ -171,125 +171,138 @@
           class="ecl-site-header-harmonised__logo-image"
           src="{{ _logo.src }}"
         />
-      </a>
-      <div class="ecl-site-header-harmonised__action">
-      {% if _login_toggle is not empty and _login_box is not empty %}
-        <div class="ecl-site-header-harmonised__login-container">
-        {% if _logged %}
-          <a
-            class="ecl-link ecl-link--standalone ecl-site-header-harmonised__login-toggle"
-            href="{{ _login_toggle.href_logged }}"
-            aria-controls="{{ _login_box.id }}"
-            data-ecl-login-toggle
-            aria-expanded="false"
-          >
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'general',
-                name: 'logged-in',
-                size: 'm'
-              },
-              extra_classes: 'ecl-site-header-harmonised__icon'
-            } only %}
-             {{- login_toggle.label_logged -}}
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'ui',
-                name: 'corner-arrow',
-                size: 'xs'
-              },
-              extra_classes: 'ecl-site-header-harmonised__login-arrow'
-            } only %}
-          </a>
-          <div
-            id="{{ _login_box.id }}"
-            class="ecl-site-header-harmonised__login-box"
-            data-ecl-login-box
-          >
-            <p class="ecl-site-header-harmonised__login-description">
-              {{- _login_box.description|raw -}}
-            </p>
-            <hr className="ecl-site-header-harmonised__login-separator" />
-            {% include '@ecl-twig/ec-component-link/ecl-link.html.twig' with {
-              link: {
-                label: _login_box.label,
-                path: _login_box.href,
-                type: "standalone"
-              }
-            } only %}
-          </div>
+        {% if group != 'group3' %}
+          {% if site_name is not empty %}
+            <div class="ecl-site-header-harmonised__site-name">{{ site_name }}</div>
+          {% endif %}
+        </a>
         {% else %}
-          <a
-            class="ecl-link ecl-link--standalone ecl-site-header-harmonised__login-toggle"
-            href="{{ _login_box.href_not_logged }}"
-          >
-            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
-              icon: {
-                path: _icon_file_path,
-                type: 'general',
-                name: 'log-in',
-                size: 'm'
-              },
-              extra_classes: 'ecl-site-header-harmonised__icon'
-            } only %}
-            {{- login_toggle.label_not_logged -}}
-          </a>
+        </a>
+        {% if site_name is not empty %}
```

Comment:
MMhh..yeah, i haven't noticed there was this difference in the two markups when i told you to re-use the existing section of the template, i'd say that to improve it a bit we could define this chunk of html as a variable when we process the variables and then print the variable, just to avoid the duplicated code.
`<div class="ecl-site-header-harmonised__site-name">{{ site_name }}</div>`

Reply:
I think in the end we just made a lot of confusion around this, @Joosthe, everything was based on the fact that you had duplicated code, now that this is a one time markup that you set, i don't see anymore the value of storing this html in a variable.